CSS/HomePg.css

```diff
@@ -0,0 +1,69 @@
+.headline {
+  background-color: #1c1a19;
+  margin: 0%;
+  background-image: url(/images/bdimage.svg);
+  background-repeat: no-repeat;
+  background-position: bottom right;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  height: auto;
+  padding: 120px 24px;
+}
+
+.mainText {
+  margin: 0%;
+  padding-top: 30px;
+  color: #ff6b00;
+}
+
+.textcolor {
+  color: #fff;
+}
+
+.contactIcons {
+  list-style: none;
+  display: flex;
+  justify-content: space-evenly;
+  padding: 0%;
+}
```

Comment:
- [ ] Kindly remove the `justify-content: space-evenly;` to make them next to each other, and center them to match the Figma design.

Your design 
![image](https://user-images.githubusercontent.com/35535130/166123647-8d3a55ca-a434-451b-9282-c37c5071c3c0.png)

Figma design
![image](https://user-images.githubusercontent.com/35535130/166123658-a8bd370d-26f2-4c4d-836c-1196f3d7887e.png)